.github/workflows/on_prerelease.yml

```diff
@@ -0,0 +1,16 @@
+name: Prerelease pipeline
```

Comment:
I would also migrate security and repolinter, I though you did already that change 🤔  I no longer see it

Reply:
I thought it had been done already, but it's not 🤦

Reply:
Done: [8590910](https://github.com/newrelic/nri-mongodb/pull/125/commits/8590910fd98002f71af8d07fd7fc6b66910c9d61)